installation_distros: update Fedora section

gotmax23 opened the pull request:
##### SUMMARY

- Mention the ansible-core and ansible-collections-* packages as well
- Replace CentOS references with Enterprise Linux. EPEL is for more than just CentOS.
- Correct links to the EPEL project documentation. We no longer use the wiki.
- Point to the official documentation for enabling the EPEL repository instead of duplicating here.
- Include maintainer contact information.

##### ISSUE TYPE
<!--- Pick one below and delete the rest -->
- Docs Pull Request

##### COMPONENT NAME
docs

##### ADDITIONAL INFORMATION
Relates: https://github.com/ansible/ansible/pull/80389

gotmax23 committed:
installation_distros: update Fedora section

- Mention the ansible-core and ansible-collections-* packages as well
- Replace CentOS references with Enterprise Linux. EPEL is for more than
  just CentOS.
- Correct links to the EPEL project documentation. We no longer use the
  wiki.
- Point to the official documentation for enabling the EPEL repository
  instead of duplicating here.
- Include maintainer contact information.

samccann commented on docs/docsite/rst/installation_guide/installation_distros.rst:
```diff
@@ -10,23 +10,45 @@ The following instructions will guide you through installing the ``ansible`` pac
 
-Installing Ansible on Fedora or CentOS 
---------------------------------------
+Installing Ansible on Fedora or Enterprise Linux
```
Enterprise linux is too loaded a term. We shouldn't include RHEL at all because that has its own installation instructions.

gotmax23 replied:
Hmm, EPEL is *Extra Packages for **Enterprise Linux***. However, I have an idea on how to rearrange this to avoid that problematic term.

samccann commented on docs/docsite/rst/installation_guide/installation_distros.rst:
```diff
@@ -10,23 +10,45 @@ The following instructions will guide you through installing the ``ansible`` pac
+for a full list of Ansible collections packaged in Fedora.
+
+
+Users of Enterprise Linux distributions,
```
```suggestion
Users of related Linux distributions,
```

samccann commented on docs/docsite/rst/installation_guide/installation_distros.rst:
```diff
@@ -10,23 +10,45 @@ The following instructions will guide you through installing the ``ansible`` pac
+
+
+Users of Enterprise Linux distributions,
+including CentOS Stream, RHEL, Almalinux, and Rocky Linux,
```
```suggestion
including CentOS Stream, Almalinux, and Rocky Linux,
```

samccann commented on docs/docsite/rst/installation_guide/installation_distros.rst:
```diff
@@ -10,23 +10,45 @@ The following instructions will guide you through installing the ``ansible`` pac
+
+Users of Enterprise Linux distributions,
+including CentOS Stream, RHEL, Almalinux, and Rocky Linux,
+can install ``ansible`` or Ansible collections from the community supported
```
```suggestion
can install ``ansible`` or Ansible collections from the community maintained
```

samccann commented on docs/docsite/rst/installation_guide/installation_distros.rst:
```diff
@@ -10,23 +10,45 @@ The following instructions will guide you through installing the ``ansible`` pac
+
+Please `file a bug <https://bugzilla.redhat.com/enter_bug.cgi>`_
+in Red Hat Bugzilla to reach the package maintainers.
 
```
Thanks for including the way to file bugs! Is this appropriate for all the fedora bits as well or only for distros using EPEL?

gotmax23 replied:
Red Hat Bugzilla is used for both Fedora Linux and Fedora EPEL.

gotmax23 committed:
distros: s/community supported/community maintained

Co-authored-by: Sandra McCann <samccann@redhat.com>

gotmax23 committed:
distros: don't name drop RHEL

Co-authored-by: Sandra McCann <samccann@redhat.com>

gotmax23 committed:
distros: split out EPEL install guide from Fedora

- Remove mentions of *Enterprise Linux* and *RHEL*
- Correctly use *Fedora Linux* (the distribution) instead of *Fedora*
  (the project as whole)
- Clarify bug report instructions

gotmax23 commented:
@samccann, see the latest force push. Let me know if this addresses your concerns. Thanks!

samccann commented:
Perfect. Thanks @gotmax23 !
